### DISCUSSION TITLE
Does tauri1.4 conflict with winres？

### DISCUSSION THREAD

**Discussion by EHW-liao** [tauri-apps/tauri#7462] [Q&A]
After upgrading the project to tauri1.4, the project was unable to start with the error code error: linking with 'link. exe' failed: exit code: 1123. After investigation, I removed the Winres dependency in cargo. toml and the build. rs related statements to successfully run it.

May I ask why this is? Is there a dependency that can replace Winres?

**Comment by FabianLars** [comment]
tauri runs winres internally as part of tauri-build (to be precise our [fork](https://github.com/tauri-apps/winres) of it.

What's your use-case here that you can't solve with what we [exposed in tauri-build](https://docs.rs/tauri-build/latest/tauri_build/struct.WindowsAttributes.html)?
